packages/gamut/src/GridForm/__tests__/GridForm-test.tsx

```diff
@@ -82,268 +97,195 @@ describe('GridForm', () => {
     ];
     const api = createPromise<{}>();
     const onSubmit = async (values: {}) => api.resolve(values);
-    render(
-      <ThemeProvider theme={theme}>
-        <GridForm
-          fields={fields}
-          onSubmit={onSubmit}
-          submit={{ type: 'fill', contents: <>Submit</>, size: 6 }}
-        />
-      </ThemeProvider>
-    );
+    const { view } = renderView({ fields, onSubmit });
 
     await act(async () => {
-      fireEvent.click(screen.getByRole('button'));
+      fireEvent.click(view.getByRole('button'));
     });
 
     // there should only be a single "assertive" error from the form submission
-    expect(await screen.queryAllByRole('alert').length).toBe(1);
-    expect(await screen.queryAllByRole('status').length).toBe(1);
+    expect(await view.queryAllByRole('alert').length).toBe(1);
+    expect(await view.queryAllByRole('status').length).toBe(1);
```

Comment:
Nit: while you're here, mind fixing these? I just noticed these don't need to `await`...

```suggestion
    expect(view.getAllByRole('alert').length).toBe(1);
    expect(view.getAllByRole('status').length).toBe(1);
```